db_sql.go

```diff
@@ -553,5 +657,21 @@ func SqlCreateSchema_User(conx *DBConnection) error {
 	versions = append(versions, `
 	ALTER TABLE authuser ADD COLUMN pwdtoken VARCHAR;`)
 
+	versions = append(versions, `
+	DROP INDEX idx_authuser_identity;
```

Comment:
Since authuser already has a field "id BIGSERIAL", I think we should reuse that, but change it's name to "userid". Although Postgres does support multiple BIGSERIAL fields per table, it is the only database which supports this concept, so I'd rather not bank on that. If we need to drop tables and recreate them, that is fine, provided of course that we preserve existing user data. The easiest way to do this is to copy existing data to temporary tables, and then copy it back once the new structure is in place. All of that should be easy to do via simple SQL commands.
Also, we may need to use a different DB migration system, so that we can make changes to the different units (session, permit, auth, user) together. This is probably going to be necessary in order to preserve existing auth data.
Now might be a good time to switch to github.com\BurntSushi\migration for migrations. It's very similar in spirit to the existing system, but it is a bit more rigorous in certain aspects.